Cookbook/Technical-Documents/AutomationIdentifiers.md

```diff
@@ -67,23 +67,90 @@ fileprivate enum NavigationBar {
 
 While this will fix the test, it is susceptible to future changes. In this case time permitting, we should attempt to define an `accessibilityIdentifier` for the element and then update the enum to use this identifier. This is not always possible as in the case with chat responses, but it is something we should strive for in order to make our tests more stable and maintainable, reducing the amount of updates needed in the future.
 
-#### Updating Queries
+### Using AccessibilityContent
+
+It is now possible to directly use accessibility identifiers defined during implementation time without resorting to raw strings.
+
+There is a namespace `AccessibilityContent` in `BabylonDependencies` that should be used by developers to define accessibility identifiers for controls, grouped by screen, for example for Promo Code Entry screen we define a `PromoCode` enum with child controls:
+
+``` swift
+public enum AccessibilityContent {
+    public enum PromoCode: String {
+        case codeCell
+    }
+}
+```
+
+After this identifier has been set on a control, we can `import BabylonDependencies` in a Screen implementation in automation, and update the query to use this identifier:
+
+``` swift
+let promofield = app.tables
+    .cells[AccessibilityContent.PromoCode.codeCell]
+    .textFields
+    .firstMatch
+```
+
+Note that the identifier is usually set on the cell itself rather than its internal components, as automation has problems finding elements otherwise.
```

Comment:
I don't think that happens always. It also probably makes sense to mention how the identifier then should be set on the cell in code.

```suggestion
Note that you may need to set the identifier on the cell itself rather than its internal components, as automation may have problems finding elements otherwise.
```